index.js

```diff
@@ -51,12 +56,16 @@ module.exports = {
     }
   },
   treeForStyles: function(){
-    var bootstrapTree = new Funnel(this.treeGenerator(path.join(this.bootstrapPath, 'stylesheets')), {
-      srcDir: '/',
+    var bootstrapTree = new Funnel(this.treeGenerator(path.join(this._bootstrapPath, 'stylesheets')), {
       destDir: '/app/styles'
     });
 
     return bootstrapTree;
-  }
+  },
 
+  treeForVendor: function() {
+    return new Funnel(this._bootstrapPath, {
```

Comment:
yeah, in theory that would be needed, but since this addon doesn't have any vendor files we should be fine without it. skipping that part also keeps the unnecessary `broccoli-merge-trees` dependency away.